Meta/Lagom/CMakeLists.txt

```diff
@@ -126,6 +126,16 @@ if (ENABLE_FUZZERS)
     add_compile_options(-fno-omit-frame-pointer)
 endif()
 
+if (ENABLE_LAGOM_COVERAGE_COLLECTION)
+    if (CMAKE_CXX_COMPILER_ID MATCHES "Clang$" AND NOT ENABLE_FUZZERS)
+        add_compile_options(-fprofile-instr-generate -fcoverage-mapping)
+        add_link_options(-fprofile-instr-generate)
+    else()
+        message(FATAL_ERROR
+            "Collecting code coverage is unsupported in this configuration.")
+    endif()
+endif()
+
```

Comment:
I wonder if we should do this in the compile_options.cmake file? lagom_options.cmake and lagom_compile_options.cmake are both loaded by Ladybird/CMakeLists.txt if you use it as the top level CMakeLists.